src/App.js

```diff
@@ -129,6 +132,7 @@ export default class App extends React.Component<{}, AppState> {
             console.log(error.name);
             console.log(error.message);
             this.setState({
+                modalIsShowing: true,
```

Comment:
Move this `modalIsShowing: true` line down 2 lines, to below `showDashConnectionError: true`. To group them together.